### PR TITLE
ci: fix ci permission issue with minikube start

### PR DESCRIPTION
**Pull request by subhamkrai**
**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: tests/scripts/github-action-helper.sh [hunk 1]
```diff
@@ -735,7 +735,7 @@ function install_minikube_with_none_driver() {
   rm "$CNI_PLUGIN_TAR"
 
   export MINIKUBE_HOME=$HOME CHANGE_MINIKUBE_NONE_USER=true KUBECONFIG=$HOME/.kube/config
-  sudo -E minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
+  minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
```
Comment (sp98) [inline tests/scripts/github-action-helper.sh:738]: why is `sudo` removed?

**Comment by parth-gr** [comment]
Will it fix 
setup cluster resources
`X Exiting due to HOST_HOME_CHOWN: Failed to change permissions for /home/runner: Error changing ownership for: /home/runner: chown /home/runner/.ghcup: no such file or directory
[2705](https://github.com/rook/rook/actions/runs/10935628958/job/30357718064?pr=14701#step:4:2711)`

File: tests/scripts/github-action-helper.sh (hunk 1 above)
Comment (subhamkrai) [reply tests/scripts/github-action-helper.sh:738]: @sp98 it was giving error
```
X Exiting due to HOST_HOME_CHOWN: Failed to change permissions for /home/runner: Error changing ownership for: /home/runner: chown /home/runner/.
``` that Parth mentioned.

**Comment by subhamkrai** [comment]
> Will it fix setup cluster resources `X Exiting due to HOST_HOME_CHOWN: Failed to change permissions for /home/runner: Error changing ownership for: /home/runner: chown /home/runner/.ghcup: no such file or directory [2705](https://github.com/rook/rook/actions/runs/10935628958/job/30357718064?pr=14701#step:4:2711)`

I think green CI indicates that it has fixed the issue

**Review by parth-gr — APPROVED** [review]

**Review by sp98 — APPROVED** [review]

**Pull request merged**